ci/build/build-pipeline.yaml

```diff
@@ -92,6 +92,16 @@ spec:
         <<: *github_source
         paths: [proxy-node-vsp-config]
 
+    - name: eidas-config
+      type: git
+      icon: bugle
+      source:
+        uri: https://github.com/alphagov/verify-eidas-config.git
+        branch: master
+        paths: [test]
```

Comment:
I ❤️this scoping to the test directory. For the deploy pipeline, we might need to separate integration and production changes a bit differently..

Reply:
My thinking was we'd have two different pipelines for integration and prod, each with their own resource definitions? If not, and they're both in the same pipeline, we could just define two resources for the different config dirs?